### PR TITLE
Add custom image operation example to docs

### PR DESCRIPTION
**Pull request by allcaps**
_Please check the following:_

-   N/A Do the tests still pass?[^1]
-   N/A Does the code comply with the style guide?
    -   YES Run `make lint` from the Wagtail root.
-   N/A For Python changes: Have you added tests to cover the new/fixed behaviour?
-   N/A For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   N/A **Please list the exact browser and operating system versions you tested**:
    -   N/A **Please list which assistive technologies [^3] you tested**:
-   YES For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

Check the docs build `topics/images.html#custom-image-filter`.

### CODE REVIEW

**Comment by gasman** [comment]
Wondering if the "extending Wagtail" section of the docs might be a better place for this?

**Review by saevarom — COMMENTED** [review]
Looks good to me. Very nice addition to the docs! 
I think @gasman makes a good point about the location of this example.

File: docs/topics/images.md [hunk 1]
```diff
@@ -414,6 +414,41 @@ Note that this will have no effect on PNG or GIF files. If you want all images t
 {% image page.photo width-400 format-webp webpquality-50 %}
 ```
 
+### Custom image filter
+
+In `wagtail_hooks.py` register your custom image filter.
+
+This example assumes Willow in combination with Pillow. If you use another image library, or like to support multiple image libraries, you need to update the code inside the run method. See the [Willow documentation](https://willow.readthedocs.io/en/latest/index.html) for more information.
```
Comment (Scotchester) [inline docs/topics/images.md:421]:
```suggestion
### Custom image filter

You can add additional image filters that aren't included in Wagtail core. To do so, add a `"register_image_operations"` hook to your `wagtail_hooks.py` file.

This assumes Willow in combination with Pillow. If you use another image library, or like to support multiple image libraries, you need to update the code inside the run method. See the [Willow documentation](https://willow.readthedocs.io/en/latest/index.html) for more information.
```

File: docs/topics/images.md [hunk 2]
```diff
@@ -414,6 +414,41 @@ Note that this will have no effect on PNG or GIF files. If you want all images t
...
+
+
+@hooks.register("register_image_operations")
+def register_image_operations():
+    return [
+        ("blur", BlurOperation),
+    ]
+```
+
+In your template:
```
Comment (Scotchester) [inline docs/topics/images.md:446]:
```suggestion
Then you can use the filter in an `{% image %}` tag in a template, like so:
```

**Review by Scotchester — COMMENTED** [review]
Thanks, @allcaps! Added a couple minor suggestions for clarity.

Question: Should we also document this hook on the [Hooks reference](https://docs.wagtail.org/en/stable/reference/hooks.html) page?

File: docs/topics/images.md (hunk 1 above)
Comment (allcaps) [reply docs/topics/images.md]: I reworded a bit more, to give some additional background. Hope you like it.

**Comment by allcaps** [comment]
See:
- `/extending/custom_image_filters.html`
- `/reference/hooks.html` (bottom of page)

I reworded a bit to better explain the Willow/Pillow situation. And to linkt the default operations vs a custom one.

Hope you like it.

**Review by lb- — APPROVED** [review]
Docs work great - copy & paste and blur!

Thanks @allcaps !

Screen Shot 2022-11-20 at 7 55 51 am

**Pull request merged**